superset-frontend/package-lock.json

```diff
@@ -10815,7 +10815,8 @@
         },
         "kind-of": {
           "version": "6.0.2",
-          "resolved": ""
+          "resolved": "https://registry.npmjs.org/kind-of/-/kind-of-6.0.2.tgz",
```

Comment:
likely due to a newer npm version in docker. We should probably maintain an `.nvmrc` version file or equivalent that can be read from docker.